### DISCUSSION TITLE
Trusted Types production support

### DISCUSSION THREAD

**Discussion by lzomlot** [jquery/jquery#5520] [General]
Our service's Trusted-Types adoption is blocked by jQuery. We are aware that you added support for Trusted-Types support v4.0 beta. Thank you for that :). What is the ETA for v4 production release that supports trusted types?

**Comment by mgol** [comment]
Closing in favor of https://github.com/jquery/jquery/discussions/5526. Also, I'm locking this one to avoid duplicate discussions.
